### PR TITLE
test(nextcloud): refactor test file structure

### PR DESCRIPTION
**Pull request by Leptopoda**
I hope the conventional commit types are right :)

### CODE REVIEW

File: packages/nextcloud/test/modles/version_check_test.dart
Comment (provokateurin) [inline packages/nextcloud/test/modles/version_check_test.dart:1]: The folder name is called `modles` but should be called `models`.

File: packages/nextcloud/test/api/notifications/notifications_helpers_test.dart [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            'type': 'type',
+            'id': 'id',
+            'delete': false,
+            'delete-all': false,
+          }),
+        );
+      });
+    });
+
+    test('to string', () {
```
Comment (provokateurin) [inline packages/nextcloud/test/api/notifications/notifications_helpers_test.dart:130]: IMO this test is a bit unnecessary, toString() will never be called in production code and rarely while developing

**Review by provokateurin — COMMENTED** [review]
LGTM, but I'm not sure if we should have webdav in lib/src/api and not in lib/src since it is very different from all the other apis. Not really related to this PR though as this was changed earlier already.

**Comment by Leptopoda** [comment]
> I'm not sure if we should have webdav in lib/src/api

I can see your point. Let's see, where #1123 and #718 go to. Feel free to change it.

File: packages/nextcloud/test/api/notifications/notifications_helpers_test.dart (hunk 1 above)
Comment (Leptopoda) [reply packages/nextcloud/test/api/notifications/notifications_helpers_test.dart:130]: I like having it to verify every attribute is also contained in the string, but I agree that it isn't necessary.
Now we have the test so :woman_shrugging:

**Comment by provokateurin** [comment]
The changes to the news test should also change the paths of the fixtures :thinking:

**Comment by Leptopoda** [comment]
We should add a CI that validates that.
I think a simple check that the repo didn't have any changes after the test would be enough?

**Comment by provokateurin** [comment]
I think just failing when a new fixture had to be created is good enough. Since we do not use retries anymore the failure should be propagated to the command.

**Review by provokateurin — APPROVED** [review]

**Pull request merged**